src/app/docs/rich-text-display/rich-text-display-docs.component.ts

```diff
@@ -0,0 +1,13 @@
+import {
+  Component
+} from '@angular/core';
+
+@Component({
+  selector: 'app-rich-text-display-docs',
+  templateUrl: './rich-text-display-docs.component.html'
+})
+export class RichTextDisplayDocsComponent {
+
+  public richText: string = `<font style="font-size: 18px" face="Arial" color="#a25353"><b>Exclusively Committed to Your Impact</b></font><p>We equip change agents with <b>cloud software</b>, <i>services</i>, <u>expertise</u>, and <font color="#a25353">data intelligence</font> designed with unmatched insight and supported with unparalleled commitment. Every day, our <b>customers</b> achieve unmatched impact as they advance their missions.</p><ul><li><a href="#">Build a better world</a></li><li><a href="#">Explore our solutions</a></li></ul>`;
```

Comment:
In the docs, this line is a pain to read as it smashes multiple lines of HTML into a single line. Can/should we insert line breaks in the logical places (i.e., at least when we switch from one element to another) to make this easier to read?

Also, I see that we are taking text from https://www.blackbaud.com/ and adding formatting to demo this component, but we excluded the first paragraph after the headline that says "Since day one, Blackbaud has been 100% focused on driving impact for social good organizations." Just my 2 cents, but without the context of that paragraph and being on our website, I think this text is initially a bit off-putting. I'd re-insert that paragraph to provide context.

Finally, our website doesn't follow the same capitalization guidance that we follow in our solutions and elsewhere, but given that this demo is directed to developers who should follow our guidance, I'd update this demo to follow our capitalization rules, not the website. You already did this with the bullet list, but I'd do it in the header too:
```suggestion
  public richText: string = `<font style="font-size: 18px" face="Arial" color="#a25353"><b>Exclusively committed to your impact</b></font><p>Since day one, Blackbaud has been 100% focused on driving impact for social good organizations.</p><p>We equip change agents with <b>cloud software</b>, <i>services</i>, <u>expertise</u>, and <font color="#a25353">data intelligence</font> designed with unmatched insight and supported with unparalleled commitment. Every day, our <b>customers</b> achieve unmatched impact as they advance their missions.</p><ul><li><a href="#">Build a better world</a></li><li><a href="#">Explore our solutions</a></li></ul>`;
```